File: Sum-Cubits-Web/src/features/reservation/models/ReservaDto.ts
import type { TurnoDto } from "@/features/turns/models/TurnoDto";
export interface ReservaDto {
  fechaReserva: string;
  listTurnsDto: TurnoDto[];
  fullNameUser: string;
  phoneUser: string;
  nameTurn: string;
  hourStart: string;
  hourEnd: string;
}

File: Sum-Cubits-Api/Sum-Cubits-Application/Features/Reservas/ReservaDto.cs
using Sum_Cubits_Application.Features.Turnos;

namespace Sum_Cubits_Application.Features.Reservas
{
    public class ReservaDto
    {
        public DateOnly FechaReserva { get; set; }
        public List<TurnoDto> ListTurnsDto { get; set; }

        public  string FullNameUser { get; set; }

        public string PhoneUser { get; set; }

        public string NameTurn { get; set; }

        public string HourStart { get; set; }
        public string HourEnd { get; set; }

    }
}
